db/seeds/questions.yml

```diff
@@ -204,26 +210,34 @@ design_portfolio:
 
 # Word limit extended for external + design/developer applicants
 
-fa21_mission_first:
-  title: If you were given the opportunity to work on any cause, what would it be and how would you work towards it? Why do you care about it?
-  hint: You don't necessarily need to have done something or had a single transformative moment — we want to know why you care, whether that's due to your experiences, perspectives, background, or anything else. This also does NOT need to relate to one of Blueprint's current or past projects, or to a cause that is directly tech-aligned. (200 words max)
+sp22_mission_first:
+  title: Talk about a social cause or problem that is important to you. What makes this cause challenging to address and why do you care about it? (200 words max)
+  hint: This also does NOT need to relate to one of Blueprint's current or past projects, or to a cause that is directly tech-aligned.
   question_category: short_answer
   question_type: long_text
   word_limit: 200
 
-fa21_perpetual_growth:
-  title: In what specific ways do you want to grow personally right now? What's motivating you? What actions do you want to take towards this growth?
-  hint: Please do not write about purely technical growth, our role descriptions, or Blueprint's mission statement. By applying, we know you want to grow in these areas. (200 words max)
+sp22_perpetual_growth:
+  title: Tell us about a time that somebody changed your mind about something you believed in or felt strongly about. How has that impacted you? (200 words max)
+  hint: Please do not write about purely technical growth, our role descriptions, or Blueprint's mission statement. By applying, we know you want to grow in these areas.
   question_category: short_answer
   question_type: long_text
   word_limit: 200
 
-fa21_always_innovate:
-  title: Tell us about a project or experience that you're proud of creating! It does not have to be technical in nature. Feel free to explain it using bullet points if you'd like.
-  hint: Don't include a link to your project in this section, there will be space for that later. Please only discuss ONE project or experience — do not treat this as a resume section. (150 words max)
+sp22_always_innovate:
+  title: Tell us about something you’re proud of creating or doing! What was the process like and what were the difficulties? What was the best part? It does not have to be technical in nature. (200 words max)
+  hint: Don't include a link to your project in this section, there will be space for that later. Please only discuss ONE thing you’re proud of — do not treat this as a resume section.
   question_category: short_answer
   question_type: long_text
-  word_limit: 150
+  word_limit: 200
+
+sp22_designer_question:
+  title: "[Designer Applicants] Please briefly describe any prior relevant experience in UI/UX, visual, or other design disciplines that might prepare you for this role. (100 words max)"
```

Comment:
Since we aren't doing the toggle version, can you change the beginning to: "[Required For Designer Applicants Only]